token/cli/src/main.rs

```diff
@@ -1384,13 +1396,13 @@ fn main() {
             SubCommand::with_name("transfer")
                 .about("Transfer tokens between accounts")
                 .arg(
-                    Arg::with_name("sender")
+                    Arg::with_name("token")
                         .validator(is_valid_pubkey)
-                        .value_name("SENDER_TOKEN_ACCOUNT_ADDRESS")
+                        .value_name("TOKEN_ADDRESS")
                         .takes_value(true)
                         .index(1)
                         .required(true)
-                        .help("The token account address of the sender"),
+                        .help("Type of token to transfer"),
```

Comment:
Nit: "Type" feels off here, but I have no better suggestion and the thesaurus is again failing me

Reply:
Do you like just "Token to transfer" any better?

Reply:
I think it's the disconnect from the code to the CLI. Where in the code we have a very explicit thing, the mint, but in CLI a very vague thing, "token". Are we trying to avoid exposing the user to mint terminology? (I've been considering renaming create-token with a similar motivation)

Reply:
Yeah, I feel you. While I personally prefer "mint" for clarity and consistency with code, it seems like the dev community understands this as "token".
I'm going to merge this with "Token to transfer" for now. We can reassess and disambiguate "token" when we (hopefully) disambiguate "account".

Reply:
```suggestion
                        .help("Token to transfer"),
```